main.tf

```diff
@@ -146,13 +146,13 @@ resource "azurerm_role_assignment" "sentinelautomationpermissions" {
 }
 #Create Sentinel Automation Rule that leverages the playbook
 resource "azurerm_sentinel_automation_rule" "emailnotifyrule" {
-  name                       = uuid()
+  name                       = uuidv5("x500", "CN=SentinelAutomationRule,ST=MD,O=CloseKnit,OU=ArchSecurity,C=US")
```

Comment:
#### Question/suggestion

- on this, perhaps a human-friendly prefix?

#### Explanation

- i switch to `uuidv5` function so what is generated is consistent instead of using a pseudo-random-generated `id`